src/components/landing-page/hero.tsx

```diff
@@ -11,12 +11,13 @@ export default ({ makeAndTrackRef, joinRef }: { makeAndTrackRef(): React.Mutable
     <p ref={makeAndTrackRef()}>
       We're on a quest to make the web more transparent
       <br />
-      and better aligned with the interests of all people
+      and better aligned with human interests
     </p>
     <div className="container" ref={makeAndTrackRef()}>
-      <LocalizedLink role="button" className="mhi-button hide-on-mobile umami--click--try-the-demo" to="/demo">
-        Try the demo
-      </LocalizedLink>
+      <a role="button" className="mhi-button hide-on-mobile umami--click--try-the-demo" target="_blank" href="https://chrome.google.com/webstore/detail/roar/jfcmnmgckhjcflmljjgjjilmjhbgdfkc?hl=en&authuser=0">
```

Comment:
No `target=_blank` as popup tabs/windows are annoying to users

Reply:
Fixed.